ui/app/components/app/home-notification/index.scss

```diff
@@ -35,9 +35,8 @@
   }
 
   &__text {
-    font-style: normal;
-    font-weight: normal;
-    font-size: 12px;
+    @include H7;
```

Comment:
This does constitute a change in line-height, but it looks better this way. I believe the notification and the tooltip were written to accommodate a wide enough range of heights for this change at least.